apps/jobs/views.py

```diff
@@ -575,3 +577,16 @@ def create_contact_message(self, request):
             return Response(serializer_class.data, status=status.HTTP_200_OK)
         else:
             return Response(serializer_class.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def list(self, request, *args, **kwargs):
+        if request.method == "GET":
+            user = request.user
+            if user.is_authenticated and user.is_moderator:
```

Comment:
what this `is_moderator` contains? boolean value? and have you anywhere in the code added any method or a variable where we can see the logic behind `is_moderator`?

Reply:
Changed the logic for moderator by checking user permissions.